backend/dal/Migrations/20200229012606_AccessRequest.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Pims.Dal.Migrations
+{
+    public partial class AccessRequest : Migration
```

Comment:
Don't need to change this, but at some point we'll need to decide whether to recreate the initial migration or create a new versioned migration.